Debugging Fundamentals/Task1/Product.cs

```diff
@@ -11,5 +11,22 @@ public Product(string name, double price)
         public string Name { get; set; }
 
         public double Price { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            if (obj == null || GetType() != obj.GetType())
+            {
+                return false;
+            }
+
+            Product other = (Product)obj;
+            return Name == other.Name && Price == other.Price;
+
```

Comment:
I think this line can be removed

Reply:
This line changed when I followed the guideline.